### DISCUSSION TITLE
Build: fatal error LNK1295: '/INCREMENTAL' not compatible with '/DEBUGTYPE' specification...

### DISCUSSION THREAD

**Discussion by FutureCompute** [dotnet/runtime#64054] [Q&A] [answered]
I downloaded .NET 6 source and ran build.cmd successfully. But when I followed method 2 of "Debugging CoreCLR on Window" in debugging.md, run "build -msbuild" I get 4 build errors as below. By following method 1 I get more build errors. I wonder if I missed anything. Thanks.

  LINK : fatal error LNK1295: '/INCREMENTAL' not compatible with '/DEBUGTYPE' specification; link without '/INCREMENTAL' [D:\runtime-6.0.0\artifacts\obj\coreclr\windows.x64.Debug\ide\tools\InjectResource\InjectResource.vcxproj]
  LINK : fatal error LNK1295: '/INCREMENTAL' not compatible with '/DEBUGTYPE' specification; link without '/INCREMENTAL' [D:\runtime-6.0.0\artifacts\obj\coreclr\windows.x64.Debug\ide\tools\GenClrDebugResource\GenClrDebugResource.vcxproj]
  LINK : fatal error LNK1295: '/INCREMENTAL' not compatible with '/DEBUGTYPE' specification; link without '/INCREMENTAL' [D:\runtime-6.0.0\artifacts\obj\coreclr\windows.x64.Debug\ide\dlls\mscordac\mscordaccore.vcxproj]
  LINK : fatal error LNK1295: '/INCREMENTAL' not compatible with '/DEBUGTYPE' specification; link without '/INCREMENTAL' [D:\runtime-6.0.0\artifacts\obj\coreclr\windows.x64.Debug\ide\dlls\mscordbi\mscordbi.vcxproj]

**Comment by huoyaoyuan** [comment, marked as answer]
Is the documentation out of date? I always used "method 1" and had no issues.

**Reply by FutureCompute** [reply]
If I use method 1 I get 30+ errors as below when build INSTALL project. So you can build INSTALL successfully?
![image](https://user-images.githubusercontent.com/38981244/157845925-b53287e9-1cff-461a-8ce4-5e7212848e5a.png)

**Reply by huoyaoyuan** [reply]
There is a known issue currently: #65562

**Comment by Sdynest** [comment]
Try to build solution using Checked configuration

**Reply by zc910704** [reply]
thanks. I used the `-c Checked` but forgot to select it  in visual studio.
